Azure/sonic-buildimage
# [201811] Update DHCPv6 counter on relayed messages

#### Why I did it
DHCPv6 Counter updates counter only on received messages, not relayed messages

#### How I did it
Update counter on messages relayed by DHCPv6 Relay

#### How to verify it
Build an image, run tests, and check the corresponding packets counter are incremented

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)

Signed off by Kelly Yeh: kellyyeh@microsoft.com
